<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSensorDisplayDigital</class>
 <widget class="QMainWindow" name="FrmSensorDisplayDigital">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(2,87, 122);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber_Value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>861</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 60, 90);
color: rgb(0, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>12345678.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>12345678</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_temperature">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>280</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 60, 90);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>12345678.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>12345678</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_tempature">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>295</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Bù nhiệt độ phép đo</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_UnitSymbol">
    <property name="geometry">
     <rect>
      <x>869</x>
      <y>60</y>
      <width>121</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 60, 90);
color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>mV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cảm biến</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1000</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(2,87, 122);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_parameter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>1001</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame_parameter{
background-color: rgb(3,87, 123);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>26</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Độ phân giải</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2,87, 122);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>26</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Đơn vị đo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>16</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_resolution">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>16</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
}
QComboBox::drop-down{
width: 20;
}
QComboBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.001</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_unit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>16</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
}

QComboBox::drop-down{
width: 20;
}
QComboBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Độ ẩm tương đối (%RH)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Độ ẩm tuyệt đối (g/m3)</string>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="toolButton_config">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>50</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Thông tin thiết bị</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(3, 87, 123);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 8px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../group.qrc">
       <normaloff>:/icon/group/iconSetup_On1.jpg</normaloff>
       <activeoff>:/icon/group/iconSetup_On.jpg</activeoff>:/icon/group/iconSetup_On1.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>3</width>
       <height>50</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton_info">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Thông tin thiết bị</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(250, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 6px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../device.qrc">
      <normaloff>:/icon/device/iconDeviceOff_401.jpg</normaloff>
      <activeoff>:/icon/device/iconDeviceOn_401.jpg</activeoff>:/icon/device/iconDeviceOff_401.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBox_tempature</tabstop>
  <tabstop>comboBox_unit</tabstop>
  <tabstop>comboBox_resolution</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../device.qrc"/>
  <include location="../group.qrc"/>
 </resources>
 <connections/>
</ui>
